### DISCUSSION TITLE
Two minor UX suggestions

### DISCUSSION THREAD

**Discussion by makeitrein** [perspective-dev/perspective#2330] [Ideas]
Hello! I've been using Perspective in my application to help display some on-chain analytics, and the reaction has been almost unanimously positive. So thank you for open-sourcing such a powerful tool!

The only two snags that I've been continuously noticing is that many of my users never figure out that the Split By + Group By + Order By + Where fields are actually dropdowns once you start typing. It might be helpful to users for the dropdown field to open on focus, not on the first key press. Perhaps it could have a chevron on the right hand side to indicate the type of interaction.

The other hickup is that the square icon didn't immediately suggest selecting and cancelling columns to many of my users. Perhaps a switch toggle or X icon might be more suggestive of the functionality?

I'd be happy to try my hand at a PR if you agree with this assessment.

**Comment by mhkeller** [comment]
I've run into both of these issues when showing people the interface. In terms of priority, the dropdown menu showing the columns would be a very helpful UX improvement.
